test-cypress/integration/setup/namespaces.spec.js

```diff
@@ -0,0 +1,352 @@
+describe('Namespaces', () => {
```

Comment:
I can easily test the pagination by importing _https://www.w3.org/TR/owl-guide/wine.rdf_ in one of the tests. WDYT ?

Reply:
Yes, do it.

Reply:
But I don't think wine.rdf has enough namespaces for multiple pages.

Reply:
It has for 2 pages

Reply:
Actually.. no :frowning_face: only 9 results after importing. Imma leave it for another time.